### PR TITLE
feat: $sendPoll , $getPoll, $getPollVotes , PERM: SEND_POLLS

### PR DESCRIPTION
**Pull request by Faf4a**
Bumps `discord.js` to `14.15.1`

Adds:
- Polls
	- SEND_POLLS (permission)
	- `$sendPoll`
	- `$endPoll`
	- `$getPollVotes`

Refactor
- `$createBanner` (use client.user.setBanner())

### CODE REVIEW

File: src/functions/message/createPoll.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+const { PollLayoutType } = require("discord.js");
+const { Time } = require("../../utils/helpers/customParser.js");
+
+module.exports = async (d) => {
+    const data = d.util.aoiFunc(d);
+    if (data.err) return d.error(data.err);
+
+    const [channelId = d.channel?.id, question, duration, allowMultiselect = "false", ...answers] = data.inside.splits;
+
```
Comment (USERSATOSHI) [inline src/functions/message/createPoll.js]: add support for json too 
with { name: name,  value:value } for ig either 1 in that we use array or u can do for each ;

**Review by USERSATOSHI — DISMISSED** [review]

**Comment by Faf4a** [comment]
All three of those need oauth to work properly, no point in having them anymore.

File: src/functions/message/createPoll.js (hunk 1 above)
Comment (USERSATOSHI) [reply src/functions/message/createPoll.js]: what i meant was for answers not for everything

**Review by USERSATOSHI — DISMISSED** [review]

File: src/functions/interaction/setApplicationCommandPermissions.js
Comment (Leref) [inline src/functions/interaction/setApplicationCommandPermissions.js:1]: Removed?

File: src/functions/interaction/removeApplicationCommandPermissions.js
Comment (Leref) [inline src/functions/interaction/removeApplicationCommandPermissions.js:1]: Removed?

File: src/functions/interaction/addApplicationCommandPermissions.js
Comment (Leref) [inline src/functions/interaction/addApplicationCommandPermissions.js:1]: Removed?

File: src/utils/Constants.js [hunk 2]
```diff
@@ -27,35 +27,24 @@ for (let i = 1; i <= 12; i++) {
...
+                    ? SI_SYMBOL[i] + "Qa"
+                    : power >= 1e9
+                      ? SI_SYMBOL[i] + "B"
+                      : power >= 1e6
+                        ? SI_SYMBOL[i] + "M"
+                        : power >= 1e3
+                          ? SI_SYMBOL[i] + "K"
+                          : SI_SYMBOL[i]
     );
 }
```
Comment (Leref) [inline src/utils/Constants.js:46]: Removed?

**Review by Leref — CHANGES_REQUESTED** [review]

**Comment by USERSATOSHI** [comment]
> All three of those need oauth to work properly, no point in having them anymore.

which ones?

**Comment by Faf4a** [comment]
> > All three of those need oauth to work properly, no point in having them anymore.
> 
> which ones?

removeappperms, addappperms, setappperms

all require oauth

**Comment by Leref** [comment]
Still need more information on the auth?

**Pull request closed without merging**